css/components/button.css

```diff
@@ -0,0 +1,89 @@
+/* === css for button component === */
+
+button{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    padding: .7rem 1.3rem;
+    font-family: inherit;
+    font-size: var(--normal-font-size);
+    font-weight: var(--font-medium);
+    border: none;
+    border-radius: 5px;
```

Comment:
Try to use same unit. If you are not resetting font size to 62.5% in root element, then divide the pixel size by sixteen and you will get the corresponding rem value

Reply:
sure will add rem instead of px units.